openstates/in/utils.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def get_with_increasing_timeout(scraper,link,fail=False,kwargs={}):
+    #if fail is true, we want to throw an error if we can't
+    #access the page we need
+    #if it's false, throw a warning and keep going
+    timeout_length = 2
+    html = None
+    while timeout_length < 65 and html is None:
+        try:
+            html = scraper.get(link,timeout=timeout_length,**kwargs)
+        except requests.exceptions.ConnectTimeout:
+            old_length = timeout_length
+            timeout_length **= 2
```

Comment:
yeah, **=2 is probably mean to future peeps and looks kind of like an emoticon (for dude with really weird facial hair and strange smile?). I'm leaving it but put a comment in :)